lib/screens/pokemon_info/widgets/tab_base_stats.dart

```diff
@@ -112,6 +115,39 @@ class _PokemonBaseStatsState extends State<PokemonBaseStats> with SingleTickerPr
     ];
   }
 
+  String _removeTrailingZero(double n){
+    String s = n.toString().replaceAll(RegExp(r"([.]*0)(?!.*\d)"), "");
+    return s;
+  }
+
+  double _getTypeEffectiveness(Pokemon pokemon, String type){
```

Comment:
this method should not belong to the `tab_base_stats.dart` too, please move it to `lib/data/pokemons.dart`